### ISSUE TITLE
Data preprocessing requests

### ISSUE THREAD

**Issue by Polaris119** [JK-the-Ko/SAR-CAM#5]
Hello! I would like to ask, would you mind updating the preprocessing code for the dataset? For example, use MATLAB to add speckle noise. There is also the "cropping of the image into a 64 × 64 patch size" mentioned in the paper. Is it a random cropping of the original image? Does this mean that the training and validation set image sizes are 64 × 64?
Looking forward to your reply.

**Comment by JK-the-Ko** [comment]
Hello,

Unfortunately, I do not have any plan for updating codes.

Image crop was only applied for training dataset where an image was sequentially cropped with a stride of 64.

For the validation dataset, please use the original image for quality assessment.

Thank you.

**Comment by Polaris119** [comment]
Thank you very much for your response! 
I have another question. In the dataset.py file, there is no mention of 'cropping' in the data augmentation section. Do you pre-crop the data and place it in the 'noisy-train-dir' and 'clean-train-dir' directories, and then run the train.py file? 
Thanks again!

```(python)
# Data Augmentation
if mode == "train" :
    self.transform = transforms.Compose([
                                                    transforms.RandomHorizontalFlip(p = p_h),
                                                    transforms.RandomVerticalFlip(p = p_v),
                                                    transforms.ToTensor()]
                                                    )

elif mode == "valid" :
    self.transform = transforms.ToTensor()
```

**Comment by JK-the-Ko** [comment]
Image cropping is done, prior. The corresponding code is only for the data augmentation. Note that "dataset.py" file is only used for training and inference.

**Comment by Polaris119** [comment]
Thank you, I understand.
